bows.js

```diff
@@ -26,7 +36,7 @@
   };
 
   var inNode = typeof window === 'undefined',
-      ls = !inNode && window.localStorage,
+      ls = !inNode && getLocalStorageSafely(),
       debugKey = ls.andlogKey || 'debug',
```

Comment:
I must admit it's been a while, and I haven't tried this, but I suspect this will just result in the exception being raised here? Do we need a better fallback for when localstorage is not available to bail from the whole module early?